c10/util/SmallBuffer.h

```diff
@@ -32,6 +32,26 @@ class SmallBuffer {
     }
   }
 
+  SmallBuffer(const SmallBuffer&) = delete;
+  SmallBuffer& operator=(const SmallBuffer&) = delete;
+
+  // move constructor is needed in function return
+  SmallBuffer(SmallBuffer&& rhs) {
+    size_ = rhs.size_;
```

Comment:
```suggestion
  SmallBuffer(SmallBuffer&& rhs) : size_{rhs.size} {
```
The above is slightly more efficient / readable.